modules/workbench/modules/reporting/main.tf

```diff
@@ -27,6 +27,7 @@ locals {
     # TODO(jaycarlton) I do not yet see a way around doing the replacement twice, as it's not possible
     #   to refer to other values in the same object when defining it.
     table_id    = replace(basename(full_path), local.TABLE_SCHEMA_SUFFIX, "")
+    range_partitioning = null
```

Comment:
Breaking change: https://github.com/terraform-google-modules/terraform-google-bigquery/blob/master/CHANGELOG.md#500-2021-03-15

* Schema should now be passed as a JSON string instead of a file path (#111)
* Table configuration must now specify `range_partitioning = null` to preserve old defaults.

Reply:
My understanding of `Table configuration must now specify range_partitioning = null to preserve old defaults.`
is we should use `range_partitioning = null`, it will preserve the old defaults because we did not use that before

Reply:
But what **are** the old defaults?

Maybe the new defaults are fine, and it's OK to remove this line?

That's what I want to know.

Reply:
the old defaults are null because we never set them. In newer terraform version, range_partitioning is now required, we need to explicitly set it to null.